src/main/webapp/app/modules/login/LoginFooter.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
@dev-fe-dx26 , Why do we need yet an other footer? Can we not use the footer I created?

Reply:
this footer is having different css..the footer which you created used background, flex and view which is not required here